package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "map_module",
+  "version": "1.0.0",
+  "description": "",
+  "main": "Gruntfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mridulnagpal/map_module.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/mridulnagpal/map_module/issues"
+  },
+  "homepage": "https://github.com/mridulnagpal/map_module#readme",
+  "devDependencies": {
+    "grunt": "^1.0.1",
+    "grunt-browserify": "^5.0.0",
+    "grunt-contrib-concat": "^1.0.1",
+    "grunt-contrib-jasmine": "^1.1.0",
+    "grunt-contrib-uglify": "^2.2.0",
+    "grunt-contrib-watch": "^1.0.0",
+    "matchdep": "^1.0.1"
+  },
+  "dependencies": {
+    "grunt-contrib-jasmine": "^1.1.0",
```

Comment:
This may need to be in `devDependencies` -- `dependencies is anything it actually includes in its own codebase.